index.html

```diff
@@ -33,28 +33,212 @@ <h1 class="headline">
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="grid-item-info">
+                        <h1 class="card-title">Multi-Post Stories Gain+Glory</h1>
+                        <div class="tech-list">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="grid-item-info">
+                        <h1 class="card-title">Multi-Post Stories Gain+Glory</h1>
+                        <div class="tech-list">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
 
+                <div class="grid-item">
+                    <div class="grid-item-info">
+                        <h1 class="card-title">Multi-Post Stories Gain+Glory</h1>
+                        <div class="tech-list">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- Kindly change the heading for each project for any heading from `h2` to `h6`, for example, `h2`. the `h1` tag should only exist once on your page
- Please change the list of technologies to be `ul` tag contains multiple `li` tags instead of `div` tag contains multiple `div` tags